dev/MRTCore/packaging/native/Microsoft.ApplicationModel.Resources.WinRt.props

```diff
@@ -16,7 +16,7 @@
   <ItemGroup Condition="'$(OutputType)' == 'Exe'">
     <Reference Include="Microsoft.ApplicationModel.Resources.winmd">
```

Comment:
><Reference Include="Microsoft.ApplicationModel.Resources.winmd"> [](start = 4, length = 64)

What about this? Is adding this reference OK?

Reply:
we need to have this reference. or we will get build break.

Reply:
won't the winmd ref be added by the nuget package install? just like the DLL refs, etc.?

Reply:
you added that. So I guess it's a question for you? :)

Reply:
currently, when the nuget package is installed, it does add the winmd ref